app/src/main/java/org/cru/godtools/fragment/BasePlatformFragment.kt

```diff
@@ -135,6 +138,9 @@ abstract class BasePlatformFragment : BaseFragment() {
             when (key) {
                 PREF_PRIMARY_LANGUAGE, PREF_PARALLEL_LANGUAGE -> loadLanguages(false)
             }
+            if (key?.startsWith(PREF_FEATURE_DISCOVERED) == true) {
```

Comment:
Do we really have to specify `== true`?

Reply:
yes because `key?.startsWith(PREF_FEATURE_DISCOVERED)` could be null. this is actually the recommended kotlin idiom when dealing with a nullable boolean

Reply:
https://kotlinlang.org/docs/reference/idioms.html#consuming-a-nullable-boolean

Reply:
Ah yes, nullable.

Reply:
matt asked a similar question on a different PR 2 days ago :)